docroot/themes/custom/mass_theme/overrides/js/iframe_resizer_parent.js

```diff
@@ -41,6 +41,7 @@ function gotResizeMessage(event) {
     if (matches[i].contentWindow === event.source) {
       // matches[i].width = Number( event.data.width )	 <--- we do not do anything with the page width for now
       matches[i].height = Number(event.data.height);
+      matches[i].setAttribute('data-resized', 1);
```

Comment:
Should we add a comment here about what this is for? Someone new to the project may not make the connection to backstop.